src/mobile/MobileToolbarBack.js

```diff
@@ -6,21 +6,24 @@ export default class MobileToolbarBack extends Component {
 
 	static propTypes = {
 		backBtnBarTitle: PropTypes.string,
+		actions: PropTypes.object,
+		to: PropTypes.string,
 	};
-
+	goBack(e) {
+		const { actions } = this.props;
+		e.preventDefault();
+		actions.resetAssetPickerFilter();
```

Comment:
resetAssetPickerFilter is existing for desktop version already. We use that to reset or refresh our asset filter or am i missing something ? CAn't we just call it and achieve the same thing for the mobile version ?